server/src/test/resources/org/opensearch/bootstrap/test.policy

```diff
@@ -10,5 +10,5 @@ grant {
   // allow to test Security policy and codebases
   permission java.util.PropertyPermission "*", "read,write";
   permission java.security.SecurityPermission "createPolicy.JavaPolicy";
-  permission java.lang.RuntimePermission "markAsSystemContext";
+  permission org.opensearch.secure_sm.ThreadContextPermission "markAsSystemContext";
```

Comment:
Do we need to set this permission here? `bootstrap/security.policy` should be enough I believe.

Reply:
This is needed to run the tests. `./gradlew :server:test --tests ThreadContextTests` would fail without this entry.

Reply:
Ok, now since you've changed `bootstrap/test-framework.policy`, this change is definitely not needed anymore